src/collection.js

```diff
@@ -231,6 +231,16 @@ class Collection {
     return !!item
   }
 
+  /**
+   * Creates an array of unique values that are included in both given array
+   * @param {Array} items

```

Comment:
Same here, can you please add an empty line before `@param`?

```suggestion
   *
   * @param {Array} items
```